api/apps/api/src/modules/projects/projects.service.ts

```diff
@@ -361,11 +359,18 @@ export class ProjectsService {
     ) {
       return left(forbiddenError);
     }
-    const project = await this.projectsCrud.create(input, info);
+    const defaultCostSurface = this.costSurfaceService.createDefaultCostSurfaceModel();
+    const project = await this.projectsCrud.create(
+      { ...input, costSurfaces: [defaultCostSurface] } as CreateProjectDTO,
+      info,
+    );
     await this.projectsCrud.assignCreatorRole(
       project.id,
       info.authenticatedUser.id,
     );
+
+    // TODO: How to handle left for Project? How is it handled by the async jobs triggered by actionAfterCreate?
```

Comment:
Since I have other pressuring items to work on, and there is still a work to do regarding including everything this PR does to the GC realm, I suggest to handle this in a separate PR